.github/workflows/e2e_test_ci.yml

```diff
@@ -272,6 +272,72 @@ jobs:
           killall spice
           cat spice.log
 
+  test_quickstart_graphql:
+      name: "Test GraphQL quickstart (${{ matrix.target.target_os }}-${{ matrix.target.target_arch }})"
+      runs-on: ${{ matrix.target.runner }}
+      needs:
+        - build
+        - setup-matrix
+
+      strategy:
+        matrix:
+          target: ${{ fromJson(needs.setup-matrix.outputs.matrix) }}
+
+      steps:
+        - uses: actions/checkout@v4
+
+        - name: download artifacts - build_${{ matrix.target.target_os }}_${{ matrix.target.target_arch }}
+          uses: actions/download-artifact@v4
+          with:
+            name: build_${{ matrix.target.target_os }}_${{ matrix.target.target_arch }}
+            path: ./build
+
+        - name: Install spice
+          uses: ./.github/actions/install-spice
+          with:
+            build-path: ./build
+
+        - name: Init spice app
+          run: |
+            spice init test_app
+
+        - name: Spice dataset configure
+          working-directory: test_app
+          run: |
+            echo -e "test_graphql_table\nrepos\ngraphql:https://api.github.com/graphql\ny" | spice dataset configure
+            # configure graphql credentials
+            echo -e "params:\n  auth_token_key: token\n  json_path: data.viewer.repositories.nodes\n  query: \"{ viewer { repositories(first: 100) { nodes { id name watchers (first: 10) { nodes { name } } } } } }\"" >> ./datasets/test_graphql_table/dataset.yaml
+            # configure env secret store
+            echo -e "secrets:\n  store: env\n" >> spicepod.yaml
+            cat spicepod.yaml
```

Comment:
E2E CI tests shouldn't depend on external services, that might block PR's. Let's replace that with local graphql app in docker.

Reply:
It runs only on workflow_dispatch event

Reply:
what is the point of that test then?

Reply:
`workflow_dispatch` covers only cases where we can't make local setup (dremio and spiceai from quickstarts)

Reply:
adding simple node.js app with graphql endpoint and some static data response should be pretty straightforward